### PR TITLE
Reworked website examples

### PR DESCRIPTION
**Pull request by radekmie**
Until now, each interactive example in the docs required us to copy its source into the `.mdx` files. This change adds a new component called `CodeSection`, which should replace all of the duplicated code with using `raw-loader` (importing source instead of the parsed file).

Additionally, I've added _section_ concept, which allows cutting only a section of the file. In the code noted as `<sectionName>`/`</sectionName>` tags.

- [x] Rewrite all examples.
- [x] Remove old `Code` component and `code` helper (`website/components/Code`).

---

```ts
type CodeSectionProps = {
  // Source language (for highlighting).
  language: string;
  // Map of patterns to replace.
  replace?: Record<string, string>;
  // Section.
  section?: string;
  // Source. Object variant is useful to work with CommonJS imports.
  source: string | { default: string };
};
```

Ping @martachrzanowska, @Monteth.

### CODE REVIEW

File: docs/examples/Tutorial/GuestFormWithErrorFields.js [hunk 1]
```diff
@@ -1,16 +1,16 @@
 import React from 'react';
-
-import GuestSchema from './GuestSchema';
 import {
   AutoForm,
```
Comment (martachrzanowska) [inline docs/examples/Tutorial/GuestFormWithErrorFields.js:3]: What about the order of imports here?

Comment (radekmie) [reply docs/examples/Tutorial/GuestFormWithErrorFields.js:3]: After rendering in `ExampleCustomizer` all `universal` imports are renamed to the correct theme (`uniforms-semantic`, etc.). I wanted to keep the correct order on the rendered docs more than in the code.

File: docs/examples/Tutorial/GuestFormWithFieldsInSchema.js [hunk 2]
```diff
@@ -1,16 +1,16 @@
 import React from 'react';
-
-import GuestSchema3 from './GuestSchema3';
 import {
   AutoForm,
```
Comment (martachrzanowska) [inline docs/examples/Tutorial/GuestFormWithFieldsInSchema.js:3]: Same as above.

File: docs/examples/Tutorial/GuestFormWithHeading.js [hunk 3]
```diff
@@ -1,16 +1,16 @@
 import React from 'react';
-
-import GuestSchema from './GuestSchema';
 import {
   AutoForm,
```
Comment (martachrzanowska) [inline docs/examples/Tutorial/GuestFormWithHeading.js:3]: Same as above.

File: docs/examples/Tutorial/GuestFormWithErrorFields.js (hunk 1 above)
Comment (martachrzanowska) [reply docs/examples/Tutorial/GuestFormWithErrorFields.js:3]: Alright :)

**Pull request merged**